Tools/src/Tools/HgcrocEmulator.cxx

```diff
@@ -46,8 +46,10 @@ bool HgcrocEmulator::digitize(
     const int &channelID,
     std::vector<std::pair<double, double>> &arriving_pulses,
     std::vector<ldmx::HgcrocDigiCollection::Sample> &digiToAdd) const {
+  // The emulation is run on each sim hit in each crossing, this is very
+  // verbose, so ldmx_log(debug) is not enough
+  bool verbose{false};
```

Comment:
Even if we have `p.logger.silence('HgcrocEmulator')`? I'm not opposed to keeping the verbose flag, but I do think we have the tools necessary to avoid it (unless there is a use-case I'm not thinking of).

Reply:
I considered this... but that means we need to put this to every cfg file, right? That might be annoying... I feel like we'd need another level below debug, `debug` shows things that are hit level, but this is a loop on the rechits, then simhits, and samples, so it's one level down from `debug` in my mind.

Reply:
And call it `spam` lol ?

Reply:
If we dont feel too funny, then `ldmx_log(diag)` ?

Reply:
Yea, in the past, I've just commented out these messages and treated it like "if you are editing this file, then these messages might be helpful to uncomment while developing"

Reply:
Made an issue: https://github.com/LDMX-Software/ldmx-sw/issues/1611
I'll keep this as it is for now